_overviews/jdk-compatibility/overview.md

```diff
@@ -10,13 +10,13 @@ Sometimes new JVM and JDK (Java Development Kit) versions require us to update S
 
 ## Version compatibility table
 
-| JDK version | Minimum Scala versions           | Recommended Scala versions                                 |
-|:-----------:|:---------------------------------|:-----------------------------------------------------------|
-| 18          | 2.13.7, 2.12.15                  | 2.13.8, 2.12.16                                            |
-| 17          | 2.13.6, 2.12.15                  | 2.13.8, 2.12.16                                            |
-| 11          | 2.13.0, 2.12.4, 2.11.12          | 2.13.8, 2.12.16, 2.11.12                                   |
-| 8           | 2.13.0, 2.12.0, 2.11.0, 2.10.2   | 2.13.8, 2.12.16, 2.11.12, 2.10.7                           |
-| 6, 7        | 2.11.0, 2.10.0                   | 2.11.12, 2.10.7                                            |
+| JDK version | Minimum Scala versions                  | Recommended Scala versions                                 |
+|:-----------:|:----------------------------------------|:-----------------------------------------------------------|
+| 18          | 2.13.7, 2.12.15, 3.0.0                  | 2.13.8, 2.12.16                                            |
+| 17          | 2.13.6, 2.12.15, 3.0.0                  | 2.13.8, 2.12.16                                            |
+| 11          | 2.13.0, 2.12.4, 2.11.12, 3.0.0          | 2.13.8, 2.12.16, 2.11.12                                   |
+| 8           | 2.13.0, 2.12.0, 2.11.0, 2.10.2, 3.0.0   | 2.13.8, 2.12.16, 2.11.12, 2.10.7                           |
+| 6, 7        | 2.11.0, 2.10.0, 3.0.0                   | 2.11.12, 2.10.7                                            |
```

Comment:
This last line is misleading. No Scala 3.x version supports JDK 6/7.

Reply:
Ah, right! I will also make sure about when 3.0.0 started supporting JDK 18

Reply:
Updated the docs based on :
https://github.com/lampepfl/dotty/blame/20d54d454bdc92edfeb44687b647a5491e35dc11/compiler/src/dotty/tools/backend/jvm/BCodeIdiomatic.scala